base/path.jl

```diff
@@ -145,7 +145,8 @@ end
 """
     dirname(path::AbstractString) -> AbstractString
 
-Get the directory part of a path.
+Get the directory part of a path. Trailing characters ('/' or '\\') in the path are
+counted as part of the path.
```

Comment:
It would not be immediately clear to me what this meant, I don't think. Can you elaborate?

Reply:
I wanted to clarify that unlike bash we do not ignore trailing slashes, e.g.
```julia
julia> dirname("/foo/bar/")
"/foo/bar"

julia> dirname("/foo/bar")
"/foo"
```

Reply:
Maybe write that and give an example?

Reply:
Better?